src/cascadia/TerminalSettingsEditor/EditColorScheme.xaml

```diff
@@ -450,6 +450,39 @@
                     </Button>
                 </StackPanel>
             </local:SettingContainer>
+
+            <local:SettingContainer x:Name="SetAsDefaultContainer"
+                                    x:Uid="ColorScheme_SetAsDefault">
+                <Button x:Uid="ColorScheme_SetAsDefaultButton"
+                        Style="{StaticResource BrowseButtonStyle}"
+                        Click="{x:Bind ViewModel.SetAsDefault_Click}"/>
+            </local:SettingContainer>
+
+            <Border MaxWidth="{StaticResource StandardControlMaxWidth}">

```

Comment:
What do we need the border for?

Reply:
So that the button gets positioned properly when the width of the window is large. Without the border, the button will be completely left aligned and the list view will be in the center.